src/components/molecules/Common/AssetModal/AssetCard/index.tsx

```diff
@@ -57,16 +59,18 @@ const Wrapper = styled(Flex)<{ selected?: boolean; cardSize?: CardSize }>`
   box-shadow: 0 6px 6px -6px ${props => props.theme.colors.other.black};
   border: 1px solid
     ${props => (props.selected ? `${props.theme.assetCard.highlight}` : "transparent")};
-  margin: ${({ cardSize }) => (cardSize === "small" ? "5px" : "10px")};
+  /* margin: ${({ cardSize }) => (cardSize === "small" ? "5px" : "10px")}; */
```

Comment:
```suggestion
```

Please remove the line if it is no longer used.

Reply:
sorry, i missed it